tests/proxy_view_tests.cpp

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <gtest/gtest.h>
+#include "proxy.h"
+#include "utils.h"
+
+namespace proxy_view_tests_details {
+
+struct TestFacade : pro::facade_builder
+    ::add_convention<pro::operator_dispatch<"+=">, void(int)>
+    ::add_convention<utils::spec::FreeToString, std::string() const>
```

Comment:
add a non-const version for `FreeToString`?

Reply:
Done